### DISCUSSION TITLE
Vote for the next feature to add

### DISCUSSION THREAD

**Discussion by progmaticltd** [progmaticltd/homebox#369] [Polls]
I will add another feature, as long as:

- It can be updated automatically, from the core Debian repositories or from another Debian compatible repo.
- Any user authentication should use the LDAP credentials.
- It does not introduce security issues.

**Comment by arodier** [comment]
The personal git server is implemented, albeit _very_ personal. No rich web interface, everything goes through ssh, using ssh certificates.

**Reply by progmaticltd** [reply]
See https://homebox.readthedocs.io/main/pages/90-other-notes/10-roadmap/
